src/platform/ESP32/KeyValueStoreManagerImpl.cpp

```diff
@@ -68,8 +68,6 @@ KeyValueStoreManagerImpl KeyValueStoreManagerImpl::sInstance;
 CHIP_ERROR KeyValueStoreManagerImpl::_Get(const char * key, void * value, size_t value_size, size_t * read_bytes_size,
                                           size_t offset_bytes)
 {
-    VerifyOrReturnError(value, CHIP_ERROR_INVALID_ARGUMENT);
```

Comment:
Please add a comment about why to remove this check.

Reply:
done